drivers/input/input_gt911.c

```diff
@@ -130,25 +129,31 @@ static int gt911_process(const struct device *dev)
 		return r;
 	}
 
-	/* obtain first point X, Y coordinates and event from:
-	 * REG_P1_XH, REG_P1_XL, REG_P1_YH, REG_P1_YL.
+	/*
+	 * obtain point X, Y coordinates and event
 	 */
-	reg_addr = REG_FIRST_POINT;
-	r = gt911_i2c_write_read(dev, &reg_addr, sizeof(reg_addr), &pointRegs, sizeof(pointRegs));
-	if (r < 0) {
-		return r;
-	}
+	if (points) {
+		if (points > 1) {
```

Comment:
this is cool but I think that most applications would not support multitouch and this would be waste of cycles for them, how about having devicetree or Kconfig option to enable or disable multitouch support?

Reply:
I agree but maybe it would be better to do it in a separate PR, what do you think?

Reply:
Thanks for the suggestion, I agree. I added a Kconfig option eventually for all input drivers to use.